[elixir] Add 1.13 , update eol dates and update latest versions

metavida opened the pull request:
This PR also makes version 1.9 marked as "Security Support **true** through the release of 1.14.0" which matches Elixir's website: https://hexdocs.pm/elixir/compatibility-and-deprecations.html

metavida committed:
Add data for Elixir 1.11 to 1.13

metavida committed:
Elixir 1.13.2 was released on 2021-12-14

Source: https://github.com/elixir-lang/elixir/blob/v1.13/CHANGELOG.md#v1132-2021-12-14

usta commented on products/elixir.md:
```diff
@@ -10,39 +10,54 @@ activeSupportColumn: true
+    release: 2021-12-03 # https://github.com/elixir-lang/elixir/releases/tag/v1.13.0
+    eol: 2024-06-01 # projected release date of 1.18.0
+    support: 2022-06-01 # projected release date of 1.14.0
+    latest: "1.13.2"
```
are you sure is this correct ? it looks like it should be 1.13.1
https://github.com/elixir-lang/elixir/tags

metavida replied:
This one was confusing to me too! I initially picked 1.13.1 as the newest because of the tags (like you linked to).

Then, as I was confirming that links worked correctly in [the PR preview](https://deploy-preview-714--endoflife-date.netlify.app/elixir) I saw that the v1.13 CHANGELOG file lists 1.13.2 with a 2021-12-14 release date! https://github.com/elixir-lang/elixir/blob/v1.13/CHANGELOG.md#v1132-2021-12-14

My Homebrew only shows 1.13.1
```
$ brew info elixir   
elixir: stable 1.13.1 (bottled), HEAD
```

asdf also only shows 1.13.1
```
$ asdf --version
v0.9.0
$ asdf list all elixir | grep 1.13 | tail   
1.13.0-rc.0-otp-23
1.13.0-rc.0-otp-24
1.13.0-rc.1
1.13.0-rc.1-otp-22
1.13.0-rc.1-otp-23
1.13.0-rc.1-otp-24
1.13.1
1.13.1-otp-22
1.13.1-otp-23
1.13.1-otp-24
```

I don't feel strongly about this (nor have in-depth knowledge of the situation) so listing 1.13.1 as the latest to be on the safe side feels fine to me 🤷

metavida replied:
Switched back to listing 1.13.1 as the latest in f61cf8bede3e8cce096ed5cc716e57b3f1105477

metavida committed:
Unclear if Elixir 1.13.2 was really released or not

It's not yet tagged in github[1], even though it's listed in the CHANGELOG[2]

[1]:https://github.com/elixir-lang/elixir/tags
[2]:https://github.com/elixir-lang/elixir/blob/v1.13/CHANGELOG.md#v1132-2021-12-14

So decided to be conservative about listing it.

usta reviewed (approved):
LGTM Thank you @metavida 🎉
